### ISSUE TITLE
Pandoc fails to build, citing missing dependency `tasty`

### ISSUE THREAD

**Issue by 8573** [NixOS/nixpkgs#52995]
## Issue description

`nixos-rebuild switch --upgrade` fails on channel `nixos-unstable` because `pandoc` fails to build:

```
[...]
Configuring pandoc-2.5...
CallStack (from HasCallStack):
  die', called at libraries/Cabal/Cabal/Distribution/Simple/Configure.hs:950:20 in Cabal-2.4.0.1:Distribution.Simple.Configure
  configureFinalizedPackage, called at libraries/Cabal/Cabal/Distribution/Simple/Configure.hs:460:12 in Cabal-2.4.0.1:Distribution.Simple.Configure
  configure, called at libraries/Cabal/Cabal/Distribution/Simple.hs:596:20 in Cabal-2.4.0.1:Distribution.Simple
  confHook, called at libraries/Cabal/Cabal/Distribution/Simple/UserHooks.hs:67:5 in Cabal-2.4.0.1:Distribution.Simple.UserHooks
  configureAction, called at libraries/Cabal/Cabal/Distribution/Simple.hs:178:19 in Cabal-2.4.0.1:Distribution.Simple
  defaultMainHelper, called at libraries/Cabal/Cabal/Distribution/Simple.hs:124:42 in Cabal-2.4.0.1:Distribution.Simple
  defaultMainWithHooks, called at Setup.hs:26:8 in main:Main
Setup: Encountered missing dependencies:
tasty >=0.11 && <1.2

builder for '/nix/store/nppdkqrsb8hk239bi6mx5y72i9z5najd-pandoc-2.5.drv' failed with exit code 1
cannot build derivation '/nix/store/xq16x6p2kkm8swzlbdbw6ccs06xvsa50-RStudio-1.1.463.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/1yjf7v05vqz8ff5rifcyqicw80xr9b1k-rdma-core-21.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/xgz9h4xb26vl7y37vz554q2fpry8qlzv-system-path.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/78dp7a9ybar87x0gfgfaq1v26ymmkpll-nixos-system-dec4a-19.03pre164581.ae002fe44e9.drv': 1 dependencies couldn't be built
error: build of '/nix/store/78dp7a9ybar87x0gfgfaq1v26ymmkpll-nixos-system-dec4a-19.03pre164581.ae002fe44e9.drv' failed
```

### Steps to reproduce

Try to build `pandoc` from current `nixos-unstable`.

## Technical details

 - system: `"x86_64-linux"`
 - host os: `Linux 4.14.59, NixOS, 19.03pre162671.44b02b52ea6 (Koi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.1.3`
 - channels(root): `"nixos-19.03pre164581.ae002fe44e9"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

**Comment by orivej** [comment]
`nixos-unstable` can advance even when the package set is pretty broken… (Related: #52929. AFAIK `nixpkgs-unstable` would not advance without `pandoc`.) `pandoc` has been fixed later, in 2534dddaa96af8faa442ba7ac360966c7990e773.

**Comment by FRidh** [comment]
pandoc has indeed been fixed
